packages/one-trust-notices/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
In other package JSON files we also have other keys, probably not strictly needed but I would follow the monorepo standards we used so far:

```
  "private": true,
  "version": "1.0.0",
  "description": "PagoPA Interoperability one trust notices job",
  "keywords": [],
  "author": "",
  "license": "Apache-2.0",
```